solutions/usaco-966.mdx

```diff
@@ -47,4 +47,46 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+
+class Usaco
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		StringTokenizer st = new StringTokenizer(br.readLine());
```

Comment:
```suggestion
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringTokenizer st = new StringTokenizer(br.readLine());
```

where does ur `br` come from? Forgot a line `BufferedReader br = new BufferedReader(new InputStreamReader(System.in))` before? 
And just throw `IOException`

Reply:
I'm sorry ,while submitting the code I missed "BufferedReader br = new BufferedReader(new InputStreamReader(System.in))".I did test the code with good number of Inputs.